cs/cs.sln

```diff
@@ -326,6 +326,13 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "grpc_generics", "..\example
 		{92915BD9-8AB1-4E4D-A2AC-95BBF4F82D89} = {92915BD9-8AB1-4E4D-A2AC-95BBF4F82D89}
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CoreNS10", "test\coreNS10\CoreNS10.csproj", "{3805B86E-2BE3-4FFA-A11C-B4981069EC88}"
```

Comment:
Since we now require VS2017, the Minimum version and what not at the top of this file should be updated. I think if you open the solution in VS2017, edit it in some way and then "Save All" it will do the right thing.